descent/targets/thermo.py

```diff
@@ -631,10 +620,77 @@ def predict(
             torch.nan if entry["std"] is None else entry["std"] * abs(type_scale)
         )
 
+        if verbose:
+            std_ref = "" if entry["std"] is None else " ± {float(entry['std']):.3f}"
```

Comment:
```suggestion
            std_ref = "" if entry["std"] is None else f" ± {float(entry['std']):.3f}"
```

Reply:
good catch, should be fixed in the latest commit!